spec/controllers/posts/create_spec.rb

```diff
@@ -44,31 +45,55 @@
       expect(flash[:error]).to eq("You do not have access to this feature.")
     end
 
-    it "requires valid dreamwidth url" do
-      login_as(user)
-      post :create, params: { button_import: true, dreamwidth_url: 'http://www.google.com' }
-      expect(response).to render_template(:new)
-      expect(flash[:error]).to eq("Invalid URL provided.")
-    end
+    context "with importer" do
+      let(:user) { create(:importing_user) }
+      let(:url) { 'http://wild-pegasus-appeared.dreamwidth.org/403.html?style=site&view=flat' }
```

Comment:
It looks like these two `let`s are the same as in the parent `context "scrape"`?

```
    let(:user) { create(:importing_user) }
    let(:url) { 'http://wild-pegasus-appeared.dreamwidth.org/403.html?style=site&view=flat' }
...
      let(:user) { create(:importing_user) }
      let(:url) { 'http://wild-pegasus-appeared.dreamwidth.org/403.html?style=site&view=flat' }
```